app/api/submission/Consignment.scala

```diff
@@ -357,75 +347,125 @@ object transportChargesType {
 
 object houseConsignmentType10 {
 
-  implicit val reads: Reads[HouseConsignmentType10] = (
-    ("1": Reads[String]) and
-      itemsPath.readArray[ConsignmentItemType09](consignmentItemType09.reads)
-  ).apply { // TODO - Should be able to change this to `(HouseConsignmentType10.apply _)` once this is all done
-    (
-      sequenceNumber,
-      ConsignmentItem
-    ) =>
-      HouseConsignmentType10(
-        sequenceNumber = sequenceNumber,
-        countryOfDispatch = None, // TODO
-        grossMass = 1, // TODO
-        referenceNumberUCR = None, // TODO
-        Consignor = None, // TODO
-        Consignee = None, // TODO
-        AdditionalSupplyChainActor = Nil, // TODO
-        DepartureTransportMeans = Nil, // TODO
-        PreviousDocument = Nil, // TODO
-        SupportingDocument = Nil, // TODO
-        TransportDocument = Nil, // TODO
-        AdditionalReference = Nil, // TODO
-        AdditionalInformation = Nil, // TODO
-        TransportCharges = None, // TODO
-        ConsignmentItem = ConsignmentItem
-      )
-  }
+  implicit val reads: Reads[HouseConsignmentType10] =
+    documentsPath
+      .read[JsArray]
+      .map(_.value.toSeq)
+      .flatMap {
+        documents =>
+          (
+            ("1": Reads[String]) and
```

Comment:
Will be fun when this "1" becomes many. 

I wonder can we make this in some way dynamic? I.e. rather than hard coding "1" use an index? As there can be only one (highlander) it would not matter anyway. 

Just trying to future proof some parts if we can.

Reply:
P.s. I am aware it was hard coded before this change so I am happy enough with the changes. Just posing the question.